File: articles/the-short-grit-scale-and-the-problem-of-measurement/cpp/short_grit_scale_score.cpp
#include <iostream>

double grit_s_score(double perseverance_effort, double consistency_interest) {
    return 0.60 * perseverance_effort + 0.40 * consistency_interest;
}

int main() {
    std::cout << "Synthetic Short Grit Scale score: "
              << grit_s_score(0.74, 0.61)
              << std::endl;
    return 0;
}

File: articles/the-short-grit-scale-and-the-problem-of-measurement/go/short_grit_scale_score.go
package main

import "fmt"

func GritSScore(perseveranceEffort float64, consistencyInterest float64) float64 {
	return 0.60*perseveranceEffort + 0.40*consistencyInterest
}

func main() {
	fmt.Printf("Synthetic Short Grit Scale score: %.3f\n", GritSScore(0.74, 0.61))
}
